FV3/gfsphysics/GFS_layer/GFS_diagnostics.F90

```diff
@@ -4310,6 +4310,19 @@ subroutine populate_emulator_diagnostics(ExtDiag, tendencies, label, nblks, idx)
       ExtDiag(idx)%data(nb)%var3 => tendencies(nb)%cloud_water
     enddo
 
+    idx = idx + 1
+    ExtDiag(idx)%axes = 2
+    ExtDiag(idx)%name = 'surface_precipitation_due_to_' // trim(label)
+    ExtDiag(idx)%desc = 'surface precipitation due to Zhao Carr ' // trim(label)
+    ExtDiag(idx)%unit = 'kg/m^2/s'
```

Comment:
Since you had `* 1000.0` in `GFS_physics_driver`, should this be g instead of kg?

Reply:
I think the units of rain1 are `m`, so `units(rain1 * density_liquid_water) = m * kg / m^3 = kg/m^2`. 1000 is the density of liquid water.